src/basic/time-util.c

```diff
@@ -120,84 +120,62 @@ usec_t map_clock_usec(usec_t from, clockid_t from_clock, clockid_t to_clock) {
         return map_clock_usec_internal(from, now(from_clock), now(to_clock));
 }
 
-dual_timestamp* dual_timestamp_from_realtime(dual_timestamp *ts, usec_t u) {
+void dual_timestamp_from_realtime(dual_timestamp *ts, usec_t u) {
```

Comment:
why?

Reply:
but it's usually what we do in similar cases. I don't really see a reason to drop this tbh